package.yaml

```diff
@@ -17,45 +17,46 @@ extra-source-files:
 
 ghc-options: -Wall
 dependencies:
-  - base >= 4.13.0.0 && < 5
-  - bytestring >= 0.10.10.1
-  - containers >= 0.6.2.1
-  - ghc >= 8.8.4
-  - list-t >= 1.0.4
-  - stringsearch >= 0.3.6.6
+  - base >= 4.8 && < 5
```

Comment:
how did you compute the new lower bounds? they don't seem to match lts-9.0 nor lts-9.21, both of which use `base-4.9.1.0`.




lts-9.21 uses base-4.9.1.0, so shouldn't this be `base >= 4.9.1.0`?

Reply:
Good catch, I think that was a copy error: I bumped it to 4.9.

Reply:
that was just one example; in the next line after that, `bytestring >=0.10.6`, also doesn't correspond to lts-9.0 nor lts-9.21, which both use `bytestring-0.10.8.1`. So, once again: how did you compute the lower bounds if you didn't take them from the lts? if you tried them manually in order to lower the bounds even more, then we should pin those versions in `oldest-supported-lts.yaml` so that CI can validate that our code continues to compile with the oldest versions of our dependencies we claim to support.

Reply:
Okay, sorry, I need to go through and recheck them all - I also don't understand... :(